src/lexer.coffee

```diff
@@ -114,12 +114,15 @@ exports.Lexer = class Lexer
       @token 'FROM', id
       return id.length
     [..., prev] = @tokens
-    forcedIdentifier = colon or prev? and
-      (prev[0] in ['.', '?.', '::', '?::'] or
-      not prev.spaced and prev[0] is '@')
-    tag = 'IDENTIFIER'
+    tag =
+      if colon or prev? and
+         (prev[0] in ['.', '?.', '::', '?::'] or
+         not prev.spaced and prev[0] is '@')
+        'PROPERTY'
+      else
+        'IDENTIFIER'
 
-    if not forcedIdentifier and (id in JS_KEYWORDS or id in COFFEE_KEYWORDS)
+    if tag == 'IDENTIFIER' and (id in JS_KEYWORDS or id in COFFEE_KEYWORDS)
```

Comment:
Thanks. Seems like I’ve gotten too used to the style guide used in another project :) I went ahead and fixed all other `==`s and `!=`s in the diff as well.